detox/src/android/espressoapi/DetoxAction.js

```diff
@@ -0,0 +1,142 @@
+/**
```

Comment:
This will be a seperate PR. The generation is currently "dump" and nothing keeps track if these functions are needed or not. It might be a little more complex and unrelated to this set of changes to remove these in the generation. But it's one of my next ups on the list